frontend/zc_messaging/src/component/ThreadList/ThreadItem/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import "./threadItem.css"
```

Comment:
instead of adding random ids to the classes use modules

Reply:
Done Sir